libraries/AP_Scripting/examples/set_message_interval.lua

```diff
@@ -0,0 +1,13 @@
+-- this script sets the update rate of mavlink messages on serial port X
+-- gcs:set_message_interval(uint8_t port_num, uint32_t msg_id, int32_t interval_us)
+--   port_num: The serial port (SerialX).
+--   msg_id: The MAVLink message ID
+--   (see https://mavlink.io/en/messages/common.html or https://mavlink.io/en/messages/ardupilotmega.html)
```

Comment:
To explain the following, I think it will be and instead of or.

ATTITUDE is https://mavlink.io/en/messages/common.html
AHRS https://mavlink.io/en/messages/ardupilotmega.html

Reply:
Thank you for the review